include/openssl/bn.h

```diff
@@ -136,14 +136,7 @@ int BN_is_word(const BIGNUM *a, const BN_ULONG w);
 int BN_is_odd(const BIGNUM *a);
 
 # define BN_one(a)       (BN_set_word((a),1))
-
-void BN_zero_ex(BIGNUM *a);
```

Comment:
Okay, I took out all changes to the header file.  This is now purely a documentation PR.  My mistake, I should have done it that way at first.